libusb/os/sunos_usb.c

```diff
@@ -831,7 +831,7 @@ sunos_check_device_and_status_open(struct libusb_device_handle *hdl,
 {
 	char	filename[PATH_MAX + 1], statfilename[PATH_MAX + 1];
 	char	cfg_num[16], alt_num[16];
-	int	fd, fdstat, mode;
+	int	fd, fdstat, mode, e;
```

Comment:
Looks like the other lines are tab aligned, and this one could be fixed up at the same time.

Reply:
Never mind, I must have seen it wrongly.